src/pp_spec.mli

```diff
@@ -0,0 +1,9 @@
+open! Stdune
```

Comment:
Could you add a comment so it's easy to figure out what this module is about?

Reply:
Done